src/box/relay.cc

```diff
@@ -905,9 +930,10 @@ relay_subscribe_f(va_list ap)
 		/* Collect xlog files received by the replica. */
 		relay_schedule_pending_gc(relay, send_vclock);
 
+		double tx_idle = ev_monotonic_now(loop()) - relay->tx_seen_time;
 		if (vclock_sum(&relay->status_msg.vclock) ==
-		    vclock_sum(send_vclock) &&
-		    ev_monotonic_now(loop()) - relay->tx_seen_time <= timeout)
+		    vclock_sum(send_vclock) && tx_idle <= replication_timeout &&
```

Comment:
Why did you replaced `timeout` with `replication_timeout`? Looks like an unintended change.

Reply:
Yep, that's a leftover from some refactoring.
Thanks for noticing! Fixed.